packages/pods-core/src/v2/pods/export/AirtableExportPodV2.ts

```diff
@@ -42,10 +43,21 @@ export class AirtableExportPodV2
 {
   private _config: RunnableAirtableV2PodConfig;
   private _airtableBase: Base;
-
-  constructor(airtable: Airtable, config: RunnableAirtableV2PodConfig) {
+  // @ts-ignore
```

Comment:
Why do we need an ignore here? seems like we are pasing in the same type as an argument to the constructor.